benchmark/Main.hs

```diff
@@ -171,18 +173,27 @@ main = do
     !lazyText <- do
         testSList <- Stream.replicateM 20 (genStrictText 50) & Stream.toList
         pure $ force $ TextL.fromChunks testSList
+    !strictByteString <- genStrictByteString 1000
+    !lazyByteString <- genLazyByteString 10 100 -- 100 Strict bytestrings each of len 10
```

Comment:
You can keep the distribution the same as the lazy text so we can compare them fairly.